src/Microsoft.ML.StandardLearners/FactorizationMachine/FieldAwareFactorizationMachineModelParameters.cs

```diff
@@ -253,11 +253,9 @@ internal void CopyLatentWeightsTo(AlignedArray latentWeights)
         /// <summary>
         /// The linear coefficients of the features. It's the symbol `w` in the doc: https://github.com/wschin/fast-ffm/blob/master/fast-ffm.pdf
         /// </summary>
-        public float[] GetLinearWeights()
+        public IReadOnlyList<float> GetLinearWeights()
```

Comment:
One line function should look like `public IReadOnlyList<float> Get() => _linearWeights`.